index.html

```diff
@@ -0,0 +1,347 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+    <link rel="stylesheet" href="style.css">
+    <title>Portifolio</title>
+</head>
+
+<body>
+    <header class="header">
+        <nav>
+            <a href="#Home" class="active"> My Logo</a>
+            <a href=" " class="icon">
+                <i class="fa-solid fa-bars fa-2x"></i>
+            </a>
+        </nav>
+    </header>
+    <section class="main-section">
+        <div class="intro-part">
+            <h1 class="intro"> I'm Dev Mike <br>
+                Glad to see<br>you!</h1>
+            <p class="about"> I'm a software developer I can help you build a project a product,feature or
+                website Look through some
+                of my work and <br> experience!If you like what you see and have a project you need coded don't hestiate
+                to contact me</p>
+            <div class="socials">
+                <h2 class="social-links">LET'S CONNECT</h2>
+                <div class="icons">
+                    <ul class="social-media">
+                        <li>
+                            <a href=""> <i class="fa-brands fa-twitter fa"></i></a>
+                        </li>
+                        <li>
+                            <a href=""> <i class="fa-brands  fa-linkedin fa "></i></a>
+                        </li>
+                        <li>
+                            <a href=""> <i class="fa-brands fa-github fa"></i></a>
+                        </li>
+                        <li>
+                            <a href=""> <i class="fa-brands fa-m fa-1x"></i></a>
+                        </li>
+                        <li>
+                            <a href=""><i class="fa-solid fa-hand-peace fa-1x"></i></a>
```

Comment:
- [ Optional] You did a great job implementing all the required changes, however, the color of your social media icons is still different from the Figma design and it can affect the look of your Portfolio. Consider changing it so that your personal Portfolio will look professional.